include/swift/AST/ASTBridging.h

```diff
@@ -382,6 +391,23 @@ class BridgedASTNode {
 #define ABSTRACT_DECL(Id, Parent) DECL(Id, Parent)
 #include "swift/AST/DeclNodes.def"
 
+// Declare `.asValueDecl` on each BridgedXXXDecl type that's also a
+// ValueDecl.
+#define DECL(Id, Parent)
+#define VALUE_DECL(Id, Parent)                                                 \
+  SWIFT_NAME("getter:Bridged" #Id "Decl.asValueDecl(self:)")                   \
+  BridgedValueDecl Bridged##Id##Decl_asValueDecl(Bridged##Id##Decl decl);
+#include "swift/AST/DeclNodes.def"
+
+// Declare `.asNominalTypeDecl` on each BridgedXXXDecl type that's also a
```

Comment:
I'm keeping this since it's needed for existing code in ASTGen after we change `BridgedEnumDecl::createParsed` return type from `BridgedNominalTypeDecl` to `BridgedEnumDecl`